src/resolvers/githubDataFetcher.js

```diff
@@ -0,0 +1,28 @@
+const fetch = require('node-fetch');
```

Comment:
Update to [ES6 modules import](http://exploringjs.com/es6/ch_modules.html). 

**tl;dr**
Instead of using require, use import

Reply:
Feito